packages/auth/tsup.config.ts

```diff
@@ -11,18 +11,11 @@ const devOpts =
     : {};
 
 export default defineConfig({
-  // explictly use CommonJS as build format here
-  // because auth package depends on @apollo/client that has no ESM support
-  format: ["cjs"],
-  entry: [
-    "src/client/index.ts",
-    "src/server/index.ts",
-    "src/core/index.ts",
-    "src/adapters/apollo",
-  ],
+  format: ["cjs", "esm"],
```

Comment:
ここはもうesmだけで良いかと思いました
```suggestion
  format: ["esm"],
```

Reply:
https://hiroppy.me/blog/nodejs-new-module-algorithm/
node@20へのバックポートとかはまだマージされてないけど時間の問題っぽいので消しちゃっていいかもですね
https://github.com/nodejs/node/pull/53500

Reply:
そしたら、消しちゃいましょう

Reply:
もう消してます